public/index.html

```diff
@@ -44,9 +44,7 @@ <h1>Real-Time Notifications</h1>
     </div>
     <script src="https://cdn.socket.io/4.7.5/socket.io.min.js"></script>
     <script>
-      // const token =
-      //   'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6NTcwMywiZW1haWwiOiJlcmljbmVtYWNoaW5lQGdtYWlsLmNvbSIsIm90cCI6IjU1NzYiLCJpYXQiOjE3MTY4ODAwOTQsImV4cCI6MTcxNjg5ODA5NH0.0-eIbOz4Ao4NtbC7xqwemrjf8mgOg-yb_1P-58M4y9I'
-      let token = prompt('Please enter your token to login', '')
+      let token = prompt('Please enter your token to login in app', '')
 
       const socket = io('http://localhost:3000', {
```

Comment:
```suggestion
      const socket = io('/', {
```
Can you check if this can work?